contracts/OriginationController.sol

```diff
@@ -471,17 +408,6 @@ contract OriginationController is
         }
     }
 
-    /**
-     * @notice Return whether the address can be used as a loan funding currency.
-     *
-     * @param token                 The token to query.
-     *
-     * @return isAllowed            Whether the contract is verified.
-     */
-    function isAllowedCurrency(address token) public view override returns (bool) {
```

Comment:
Were these removed for contract size reasons? These are useful and if so we should make a note to reintroduce them to a singleton `OriginationWhitelist` contract (as discussed in standup this morning)

Reply:
Yeah these will be added back in the next PR where a OC singleton contract is added and multiple OCs can be supported.